Cellpose_Plugin_CellProfiler_Instructions/README.md

```diff
@@ -0,0 +1,97 @@
+# Cellpose Plugin for CellProfiler installation
+
+## For Linux:
```

Comment:
Because Linux instructions are the only ones ready with this PR, it may make sense to replace the sub-header with a minor note instead.

```suggestion
__Please note__: instructions below presume a Linux environment.
```

Reply:
True! I have implemented this.